qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -151,7 +151,7 @@
                     type = x86_cpu_test_0x40000001
                     start_vm = yes
                     test_dir = '/home'
-                    source_file = 'cpuid.src.tar.gz'
+                    source_file = 'cpuid.x86_64.tar.gz'
```

Comment:
It's a single binary file, I don't think we need to compress it.

Reply:
> It's a single binary file, I don't think we need to compress it.

As we talked offline I updated and thanks.